openmetadata-ui/src/main/resources/ui/src/components/MyData/LeftSidebar/left-sidebar.less

```diff
@@ -271,3 +271,50 @@
   background-color: @white;
   overflow-x: hidden;
 }
+
+// support for rtl in sidebar
+
+.left-sidebar-col-rtl {
+  .left-sidebar-container {
+    right: 0;
+    left: initial;
+  }
+
+  .brand-logo-container {
+    padding-left: initial;
+    padding-right: 14px;
+  }
+
+  .left-sidebar-menu.ant-menu {
+    .ant-menu-item {
+      padding: 8px 13px 8px 0 !important;
```

Comment:
Can we avoid `!important`?

Reply:
antd items using inline styles for padding and we can only override them using important here and other places